core/chains/evm/txmgr/test_helpers.go

```diff
@@ -145,7 +143,7 @@ func (c *MockConfig) FinalityTagEnabled() bool          { return c.finalityTagEn
 func (c *MockConfig) RPCDefaultBatchSize() uint32       { return c.RpcDefaultBatchSize }
 
 func MakeTestConfigs(t *testing.T) (*MockConfig, *TestDatabaseConfig, *TestEvmConfig) {
-	db := &TestDatabaseConfig{defaultQueryTimeout: pg.DefaultQueryTimeout}
+	db := &TestDatabaseConfig{defaultQueryTimeout: 10 * time.Second}
```

Comment:
should this be a named constant living in common?

Reply:
I don't think that we need to couple all of the modules together, but this does seem worth a constant somewhere within EVM.

Reply:
Added constant to `evm/txmgr/utils`